Enable/Utilities/Utilities.m

```diff
@@ -223,12 +225,18 @@ + (void) getLocationFromPOI_idStr: (NSString * _Nonnull) POI_idStr withCompletio
 }
 
 + (void) getLocationsFromLocation: (CLLocationCoordinate2D) location corner: (CLLocationCoordinate2D) corner withCompletion: (void (^_Nonnull)(NSArray<Location *> * _Nullable locations, NSDictionary * _Nullable error))completion{
-
     PFQuery * query = [PFQuery queryWithClassName:@"Location"];
     PFGeoPoint * farRightCorner = [PFGeoPoint geoPointWithLatitude:corner.latitude longitude:corner.longitude];
     PFGeoPoint * point = [PFGeoPoint geoPointWithLatitude:location.latitude longitude:location.longitude];
-    float radius = [point distanceInMilesTo:farRightCorner];
-    if(radius > 50) return;
+    double radius = [point distanceInMilesTo:farRightCorner];
+    if(radius > kMaxRadius) {
+        return;
```

Comment:
why do you return here? Shouldn't you set the radius to kMaxRadius and keep the execution going?